instrumentation/opentelemetry-instrumentation-fastapi/tests/test_fastapi_instrumentation.py

```diff
@@ -236,6 +236,7 @@ def test_basic_metric_success(self):
             "net.host.port": 443,
             "http.status_code": 200,
             "http.target": "/foobar",
+            "http.route": "/foobar",
```

Comment:
https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2624#issuecomment-2200646062

Note that here `http.target` and `http.route` are equal.